config/initializers/assets.rb

```diff
@@ -59,6 +59,8 @@
 Rails.application.config.assets.precompile += %w(comments.js)
 Rails.application.config.assets.precompile += %w(projects/show.js)
 Rails.application.config.assets.precompile += %w(projects/introdutory_popup.js)
+Rails.application.config.assets.precompile += %w(step_result_comments.js)
```

Comment:
This file is actually not present. It seems it was removed in some recent commits, so it doesn't make sense to include it here...